lib/notify_user/channels/action_mailer/action_mailer_channel.rb

```diff
@@ -12,14 +12,24 @@ def default_options
       }
     end
 
-    def deliver(notification, options={})
+    def deliver(notification_id, options={})
+      if notification_id.is_a? NotifyUser::BaseNotification
+        raise RuntimeError, "Must pass notification ids, not the notification itself"
+      end
+
+      notification = NotifyUser::BaseNotification.find(notification_id)
+
       NotifyUser::NotificationMailer.notification_email(notification, default_options.deep_merge(options)).deliver
     end
 
-    def deliver_aggregated(notifications, options={})
+    def deliver_aggregated(notification_ids, options={})
+      if notification_ids.first.is_a? NotifyUser::BaseNotification
+        raise RuntimeError, "Must pass notification ids, not the notifications themselves"
+      end
+
+      notifications = NotifyUser::BaseNotification.where(id: notification_ids)
+
       NotifyUser::NotificationMailer.aggregate_notifications_email(notifications, default_options.deep_merge(options)).deliver
     end
```

Comment:
Haha think this is actually your change from the GCM branch. So sure 👍 

But again...this falls into "how do we want to send aggregated notifications" territory. If params merging is the go going forward, we don't actually need this method.